src/components/dictionaries/merriamwebster/View.tsx

```diff
@@ -1,61 +1,140 @@
+  's',
+  't',
+  'u',
+  'v',
+  'w',
+  'x',
+  'y',
+  'z'
+]
+
+export const DictMerriamWebster: FC<ViewPorps<MerriamWebsterResultV2>> = ({
   result
 }) => (
-  <ul className="dictMerriamWebster-List">
-    {result.map((def, defI) => (
-      <li key={def.meaning} className="dictMerriamWebster-Item">
-        <div className="dictMerriamWebster-TitleBox">
-          <sup className="dictMerriamWebster-Sup">{defI + 1}</sup>
-          <span className="dictMerriamWebster-Title">{def.title}</span>
-          <span className="dictMerriamWebster-Pos">{def.pos}</span>
-          <Speaker src={def.pron} />
+  // <ul className="mw-list">
+  <ul>
+    {result.groups.map((g, i) => (
+      <li key={`${`mw-g`}-${i}`} className="mw-item">
+        <div className="mw-top-container">
+          <div className="mw-title-area">
+            {/* <sup className="mw-Sup">{i + 1}</sup> */}
+            <span className="mw-title">{g.title}</span>
+            <span className="mw-pos">({g.pos})</span>
+          </div>
+          <div className="mw-prs">
+            {g.pr?.syllable && (
+              <span className="mw-syllable">{g.pr?.syllable}</span>
+            )}
+            {g.pr?.phonetics &&
+              g.pr?.phonetics.map((v, j) => (
+                <div
+                  key={'mw-pt-' + j}
+                  className={v.audio ? 'mw-pt' : 'mw-pt-text'}
+                >
+                  {v.symbol}
+                  {v.audio && <Speaker src={v.audio} />}
+                </div>
+              ))}
+          </div>
         </div>
 
-        <div className="dictMerriamWebster-PronBox">
-          {def.syllables && (
-            <>
-              <span className="dictMerriamWebster-Syllables">
-                {def.syllables}
-              </span>
-              <span className="dictMerriamWebster-Break">|</span>
-            </>
-          )}
-          {def.pr && <span>\{def.pr}\</span>}
-
-          {def.headword && (
+        {g.sections.map((s, n) => (
+          <div key={'mw-section-' + n} className="mw-section">
+            {s.title && <div className="mw-section-title">{s.title}</div>}
             <div>
-              <StrElm
-                tag="p"
-                className="dictMerriamWebster-Headword"
-                html={def.headword}
-              />
+              {s.meaningGroups.map((means, o) => (
+                <div key={'mw-mg-' + o} className="mw-mg-area">
+                  <div className="mw-mg-left">
+                    <div className="mw-mg-sign"> {o + 1}</div>
+                    <div className="mw-mg-line"></div>
+                  </div>
+
+                  <div className="mw-mg-right">
+                    {means.map((mean, k) => (
+                      <div key={'mw-meaning-' + k} className="mw-mean-area">
+                        {(mean.examples || mean.explaining) &&
+                          means.length > 1 && (
+                            <span className="mw-mean-sign">{alphabet[k]}</span>
```

Comment:
Can `alphabet[k]` be replaced with `String.fromCharCode(k - 97)`?

Reply:
thanks for point it out, done.